content/projects/gsoc/2020/application.md

```diff
@@ -20,13 +20,14 @@ Please feel free to contribute to it by proposing pull requests against the page
 
 ### Organization Profile
 
+* Website URL: http://jenkins.io/
 * Tagline (up to 80 symbols): Open-source automation server for building great things at any scale
 * Technology tags (max 5):
   * java
-  * groovy
```

Comment:
Since `groovy` will be removed can we add `machine learning`?

Reply:
Go was added. But indeed we should review the final list of tags